### ISSUE TITLE
Issue with README

### ISSUE THREAD

**Issue by tjmcewan** [deragon/autopoweroff#24]
Heya, thanks for this project. :)

The README repeats the Host parameter comment for `CpuPercentage` so doesn't give an indication of how to use that setting.

I'll go digging through the code. :)

Thanks again!

**Comment by tjmcewan** [comment]
Also, how do I start it on ubuntu server?  I have installed the deb package but nothing happens.  I've rebooted, still nothing.  There's no process running, no entries in syslog, no `/var/log/autopoweroff.log` file.

**Comment by deragon** [comment]
Greetings.  You are right, the documentation is deficient.  I have to update it.   Following, the commands to get Autopoweroff working at startup:

```
systemctl enable autopoweroff  # Enable Autopoweroff at every reboot.
systemctl start autopoweroff   # Start Autopoweroff immediately.  
```

**Comment by tjmcewan** [comment]
A couple of other things I noticed:

- the anchor links to the config file section in the readme don't seem to work?
- the file location is given as `/etc/autopoweroff.conf` but (at least my system) the config file is being read from `/etc/autopoweroff/autopoweroff.conf`

**Comment by deragon** [comment]
Fixed with commit 2c17789d293ffc2f39c5ca5d5b0ebe721c6e2c22.
